modules/install-osp-deploy-dualstack.adoc

```diff
@@ -24,15 +24,14 @@ DHCP must be enabled on the subnets.
 . Add the IPv6 subnet to the router to enable router advertisements. If you are using a provider network, you can enable router advertisements by adding the network as an external gateway, which also enables external connectivity.
 
 
-. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.
-
-.Example `install-config.yaml`
-
+. To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file.
++
+To configure IPv4 and IPv6 address endpoints for cluster nodes, edit the `install-config.yaml` file. The following is an example of an `install-config.yaml` file.
```

Comment:
do you need a dot int the beginning of this sentence?

Reply:
fixed :) I had an extra repeated line